### DISCUSSION TITLE
Can you execute Tr3nch via rigtools/

### DISCUSSION THREAD

**Discussion by lo1746** [3kh0/ext-remover#1184] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I've seen this exploit called tr3nch and Im wondering if you can somehow execute the script itself in rigtools. My chromeos version is 128.

**Comment by Perplexedykwhat** [comment]
no. can't execute scripts on a chrome url.

**Comment by S-PScripts** [comment]
closed as inactive
